packages/runtime/src/templates/edge-shared/utils.ts

```diff
@@ -31,8 +31,13 @@ export const addMiddlewareHeaders = async (
   // We need to await the response to get the origin headers, then we can add the ones from middleware.
   const res = await originResponse
   const response = new Response(res.body, res)
+  let originCookies = response.headers.get('set-cookie')
```

Comment:
Because `originCookies` isn't being modified after being set, this can instead be written as

```suggestion
  const originCookies = response.headers.get('set-cookie')
```